lib/map.js

```diff
@@ -186,7 +186,11 @@ Mapper.prototype.push = function (data) {
   if (! this.isMulti() || data === null) {
     return push.call(this, data)
   } else {
-    return data.every(push.bind(this))
+    var needMoreData = false
+    for (var i = 0; i < data.length; ++i) {
+      needMoreData = push.call(this, data[i])
```

Comment:
I kind of feel this should be either `needMoreData = needMoreData && push.call()` or `needMoreData = needMoreData || push.call()`?